client/components/catalog/Card/Tags/index.vue

```diff
@@ -7,7 +7,7 @@
         @click:close="showDeleteConfirmation(id, name)"
         color="blue-grey lighten-4"
         label light small close
-        class="mr-2">
+        class="readonly mr-2">
```

Comment:
These ones are not readonly (actionable)

Reply:
Correct me if I'm wrong. My view of it was that if a chip is not clickable it should not have `readonly` class. A chip may contain clickable actions, but the chip itself is not clickable. 
For example, let's look at these two chips:
This chip is not clickable but it contains clickable actions:
![Screenshot 2020-11-24 at 11 01 17](https://user-images.githubusercontent.com/26693037/100079525-08314580-2e45-11eb-88d5-1ac0e03af707.png)
This chip is clickable and it doesn't have `readonly` class:
![Screenshot 2020-11-24 at 11 01 55](https://user-images.githubusercontent.com/26693037/100079577-1bdcac00-2e45-11eb-94be-b838bef8a47b.png)

Reply:
You have documentation here for each usage type (Right sidebar)
https://material.io/components/chips#input-chips

Reply:
@underscope Thanks for your comment.
All chips in the MD specification have a hover effect, so I wasn't really sure is that relevant. 
Here, in MD implementation: https://material.angular.io/components/chips/examples even basic chips got a hover effect.
I'm going to fix this and remove the `readonly` class from actionable chips 👍